packages/hippodrome-ui/src/components/SplashScreens.tsx

```diff
@@ -0,0 +1,107 @@
+import { Heading, Text, VStack, Box, Grid, Image } from "@chakra-ui/react"
+import React from "react"
+import "@fontsource/zen-dots"
+import lightRectImage from "../assets/light-rect.svg"
+import coinsImage from "../assets/coins.svg"
+import ConnectButton from "./ConnectButton"
+
+interface SplashScreenProps {}
+
+const SplashScreen: React.FC<SplashScreenProps> = () => {
+  const features = [
+    {
+      icon: lightRectImage,
```

Comment:
I think we should replace this. Maybe just a coin since the other one is a deposit.